<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>890</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Curseur Midi</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="ongletsPrincipaux">
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="systemTab">
       <attribute name="title">
        <string>System</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="maxClientsLabel">
          <property name="text">
           <string>Nombre de clients maximum : </string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="maxClientsLineEdit"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="maxPortsLabel">
          <property name="text">
           <string>Nombre de ports maximum : </string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="maxPortsLineEdit"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="currentClientsLabel">
          <property name="text">
           <string>Nombre de clients actuels : </string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="currentClientsLineEdit"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="maxChannelsLabel">
          <property name="text">
           <string>Nombre de canaux maximum : </string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="maxChannelsLineEdit"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="clientTab">
       <attribute name="title">
        <string>Client</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="1">
           <widget class="QPushButton" name="openClientButton">
            <property name="text">
             <string>Open Client</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QPushButton" name="closeClientButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Close Client</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QLineEdit" name="clientNameLineEdit">
            <property name="text">
             <string>*** Sans nom ***</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="clientNumberLabel">
            <property name="text">
             <string>Numéro du client : </string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLineEdit" name="clientNumberLineEdit">
            <property name="text">
             <string>Closed!!!</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="clientNameLabel">
            <property name="text">
             <string>Nom du client : </string>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="portTab">
       <attribute name="title">
        <string>Port</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="portsLayout">
          <item>
           <layout class="QVBoxLayout" name="inputPortLayout">
            <item>
             <widget class="QLabel" name="inputPortLabel">
              <property name="text">
               <string>Entrée</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="numberInputPortLayout">
              <item>
               <widget class="QLabel" name="numberInputPortLabel">
                <property name="text">
                 <string>Nombre d'entrée : </string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="numberInputPortLineEdit">
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QComboBox" name="inputPortComboBox"/>
            </item>
            <item>
             <layout class="QHBoxLayout" name="buttonsInputPortLayout">
              <item>
               <widget class="QPushButton" name="createInputPortButton">
                <property name="text">
                 <string>Créer un nouveau port d'entrée</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="killInputPortButton">
                <property name="text">
                 <string>Supprimer le port sélectionnée</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="outputPortLayout">
            <item>
             <widget class="QLabel" name="outputPortLabel">
              <property name="text">
               <string>Sortie</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="numberOutputPortLayout">
              <item>
               <widget class="QLabel" name="numberOutputPortLabel">
                <property name="text">
                 <string>Nombre de sortie : </string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="numberOutputPortLineEdit">
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QComboBox" name="outputPortComboBox"/>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QPushButton" name="pushButton">
                <property name="text">
                 <string>Créer un nouveau port de sortie</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_2">
                <property name="text">
                 <string>Supprimer le port sélectionné</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="curseurTab">
       <attribute name="title">
        <string>Curseur</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLCDNumber" name="lcdNumber">
          <property name="lineWidth">
           <number>1</number>
          </property>
          <property name="smallDecimalPoint">
           <bool>true</bool>
          </property>
          <property name="digitCount">
           <number>3</number>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QSlider" name="curseur">
            <property name="maximum">
             <number>127</number>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksBothSides</enum>
            </property>
            <property name="tickInterval">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="quitButton">
        <property name="text">
         <string>Quitter</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>890</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>curseur</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>457</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>340</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
